luigi/static/visualiser/js/visualiserApp.js

```diff
@@ -504,9 +504,6 @@ function visualiserApp(luigi) {
 
         $('input[name=vis-type]').on('change', function () {
             changeState('visType', $(this).val());
-
-            initVisualisation(visType);
```

Comment:
I think both `initVisualization` and `updateVisType` should be invoked here. Maybe this way:

```javascript
var visType = $(this).val();
changeState('visType', visType);
initVisualisation(visType);
updateVisType(visType);
```